CardIO_Public_API/CardIOPaymentViewController.h

```diff
@@ -76,6 +76,13 @@
 /// Defaults to NO.
 @property(nonatomic, assign, readwrite) BOOL suppressScannedCardImage;
 
+/// After a successful scan, the CardIOView will briefly display an image of the card with
+/// the computed card number superimposed. This property controls how long (in seconds)
+/// that image will be displayed.
+/// Set this to 0.0 to suppress the display entirely.
+/// Defaults to 1.0.
```

Comment:
Looks like the default for a `CardIOView` is indeed `1.0`. But for a `CardIOPaymentViewController` the default was formerly `0.1` (from the [line below that you're (correctly) changing](https://github.com/card-io/card.io-iOS-source/pull/29/files?diff=unified#diff-9035a9777252dfa6f23022d2f8e5c87eL101).

We should maintain those respective defaults. For `CardIOView` this is the only visible feedback, while for `CardIOPaymentViewController` this is only a momentary pause before animating over to the data-entry view.